src/style.css

```diff
@@ -0,0 +1,121 @@
+:root {
+  --main-accent-color: #060b35;
+  --secondary-color: #616070;
+  --color-of-button: #00acdc;
+}
+
+* {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  box-sizing: border-box;
+}
+
+.product-card {
```

Comment:
Create separate files for card and stars BEM block and move styles for them there